edge-functions/sveltekit-redirect/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "sveltekit-redirect",
+	"version": "0.0.1",
+	"scripts": {
+		"dev": "svelte-kit dev",
+		"build": "svelte-kit build",
```

Comment:
Should this be `vc dev` and `vc build`?

Right now middleware just doesn't work in development; `svelte-kit dev` doesn't understand Vercel middleware. That may change in future (via an expanded adapter API), but in the meantime I'm hoping that `vc dev` will allow it to Just Work in a future version. We should check though, because otherwise this example is DOA :)